Userland/Libraries/LibGfx/ImageFormats/WebPWriterLossless.cpp

```diff
@@ -43,6 +35,184 @@ NEVER_INLINE static ErrorOr<void> write_image_data(LittleEndianOutputBitStream&
     return {};
 }
 
+struct code_length_symbol {
```

Comment:
Shouldn't that be named `CodeLengthSymbol`?

Reply:
This is the name it has over in Deflate.cpp, probably because that code is based on https://www.hanshq.net/zip.html and that's the name there.

Reply:
Sure, done. As I said, I don't think the name is different because it's years-old, but to look a bit like the code in https://www.hanshq.net/zip.html, which is what the LibCompress code is heavily based on.